pkg/koordlet/statesinformer/callback_runner.go

```diff
@@ -104,7 +106,7 @@ func (s *callbackRunner) SendCallback(objType RegisterType) {
 			klog.Infof("last callback runner %v has not finished, ignore this time", objType.String())
```

Comment:
actually, it is not necessary to maintain multi callback events, since each callback will always get the latest obj after event processed.